StyleCop.Analyzers/Directory.Build.targets

```diff
@@ -1,17 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
 
+  <PropertyGroup>
+    <SkipAddAllResxFilesAsAdditionalFiles>true</SkipAddAllResxFilesAsAdditionalFiles>
```

Comment:
📝 I wasn't able to get this working under the default configuration, which is a bad sign for usability

Reply:
➡️ This is finally not needed with the recent version of the source generator